src/pe_reports/metrics.py

```diff
@@ -29,40 +31,31 @@ def __init__(self, trending_start_date, start_date, end_date, org_uid):
             org_uid, trending_start_date, end_date
         )
         self.creds_view = query_creds_view(org_uid, start_date, end_date)
+        self.creds_by_day = query_credsbyday_view(
+            org_uid, trending_start_date, end_date
+        )
+        self.breach_details_view = query_breachdetails_view(
+            org_uid, start_date, end_date
+        )
 
-    def by_days(self):
+    def by_week(self):
         """Return number of credentials by day."""
```

Comment:
Does this docstring need to change, since the name and content of the function has changed?

Reply:
Yes, thanks for catching that. Fixed here: https://github.com/cisagov/pe-reports/pull/191/commits/241889fb241ffb415d1aae781359904beef55893

Reply:
@dav3r issue addressed in the following commit: [241889f](https://github.com/cisagov/pe-reports/pull/191/commits/241889fb241ffb415d1aae781359904beef55893)